FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/demo_cloudformation_template.json

```diff
@@ -0,0 +1,217 @@
+{
+    "Parameters": {
+        "CertificateId": {
+            "Type": "String",
+            "Description": "Enter the \"certificateId\" of the certificate created using the \"aws iot create-keys-and-certificate\" command.\nFor more information, see the documentation: https://freertos.org/iot-fleet-provisioning/demo.html"
```

Comment:
Can this also be created by the template?

Reply:
Unfortunately, CloudFormation is unable to create keys/certificates.